### PR TITLE
Updated alibi detect to 0.4.3

### LINKED ISSUE #2667: Update metrics exposed by alibi detect server to include all newer components (threshold, etc)
In the latest version of alibi detect (0.4.3) there are new metrics like threshold that would be useful to expose as metrics, which would require both exposing them as well as upgrading the version of the alibi detect server (Which would also require upgrading all relevant examples)

### PR DESCRIPTION
**Pull request by axsaucedo**
Fixes #2667 

* Updated alibidetect image to use alibi detect 0.4.3
* Retrained cifar10_ks_cd model to 0.4.3
* Removed p_values from metrics given the detail is not currentl used
* Tested cifar10 notebook using new image
* Updated makefile to work with 0.4.3 model
* Tested outlier detector from makefile working still using 0.4.3 without changes

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **axsaucedo**
You can assign the PR to them by writing `/assign @axsaucedo` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Thu Nov 19 20:02:04 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2678/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2678 --build=1

**Comment by seldondev** [comment]
Thu Nov 19 20:02:36 UTC 2020
The logs for [pr-build] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2678/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2678 --build=3

**Comment by seldondev** [comment]
Thu Nov 19 20:03:09 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2678/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2678 --build=2

**Comment by seldondev** [comment]
Thu Nov 19 20:03:20 UTC 2020
The logs for [pr-build] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2678/5.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2678 --build=5

**Comment by seldondev** [comment]
Thu Nov 19 20:03:56 UTC 2020
The logs for [lint] [6] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2678/6.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2678 --build=6

**Comment by seldondev** [comment]
Thu Nov 19 20:04:24 UTC 2020
The logs for [lint] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2678/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2678 --build=4

**Comment by seldondev** [comment]
Fri Nov 20 10:26:47 UTC 2020
The logs for [lint] [8] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2678/8.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2678 --build=8

**Comment by seldondev** [comment]
Fri Nov 20 10:27:00 UTC 2020
The logs for [pr-build] [7] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2678/7.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2678 --build=7

**Comment by seldondev** [comment]
Fri Nov 20 10:37:37 UTC 2020
The logs for [lint] [10] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2678/10.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2678 --build=10

**Comment by seldondev** [comment]
Fri Nov 20 10:37:52 UTC 2020
The logs for [pr-build] [9] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2678/9.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2678 --build=9

File: components/alibi-detect-server/adserver/cd_model.py
Comment (arnaudvl) [inline components/alibi-detect-server/adserver/cd_model.py:27]: No "feature_score" or "batch_score" fields anymore, so unnecessary metrics logging.

File: components/alibi-detect-server/adserver/cd_model.py
Comment (arnaudvl) [inline components/alibi-detect-server/adserver/cd_model.py:40]: *distance* type can either be **float** or **np.ndarray**, so proper type needs to be picked up and logged. Iterate through array and append or add float.

File: components/alibi-detect-server/adserver/cd_model.py
Comment (arnaudvl) [inline components/alibi-detect-server/adserver/cd_model.py:52]: See comment on *distance* type. Same thing.

File: components/alibi-detect-server/adserver/cd_model.py
Comment (arnaudvl) [inline components/alibi-detect-server/adserver/cd_model.py:37]: *is_drift* type is either **int** or **np.ndarray** (in the case of feature-wise drift for the KS detector). Needs to support both (optionally iterate through array).

**Comment by seldondev** [comment]
Fri Nov 20 13:00:11 UTC 2020
The logs for [pr-build] [11] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2678/11.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2678 --build=11

**Comment by seldondev** [comment]
Fri Nov 20 13:00:09 UTC 2020
The logs for [lint] [12] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2678/12.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2678 --build=12

**Comment by seldondev** [comment]
Fri Nov 20 13:10:01 UTC 2020
The logs for [pr-build] [13] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2678/13.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2678 --build=13

**Comment by seldondev** [comment]
Fri Nov 20 13:10:07 UTC 2020
The logs for [lint] [14] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2678/14.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2678 --build=14

**Review by arnaudvl — APPROVED** [review]

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[arnaudvl](https://github.com/SeldonIO/seldon-core/pull/2678#pullrequestreview-535482279 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **axsaucedo**
You can assign the PR to them by writing `/assign @axsaucedo` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by SachinVarghese — COMMENTED** [review]
/lgtm

**Comment by seldondev** [comment]
failed to trigger Pull Request pipeline
* failed to create agent
* failed to calculate in repo config
* failed to load trigger config for repository SeldonIO/seldon-core for ref d7b7b4ffae050d54bc3c168d12e99e1bc9cf2ce3
* failed to find any lighthouse configuration files in repo SeldonIO/seldon-core at sha d7b7b4ffae050d54bc3c168d12e99e1bc9cf2ce3
* failed to process repo SeldonIO/seldon-core refref d7b7b4ffae050d54bc3c168d12e99e1bc9cf2ce3
* failed to list files in directory /var/tmp/gitrepo660600678/.lighthouse
* open /var/tmp/gitrepo660600678/.lighthouse
* no such file or directory

**Pull request merged**
